backend/EasySpeak.Notifier/EasySpeak.Notifier.WebAPI/Services/ConsumerHostedService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using EasySpeak.RabbitMQ;
+using EasySpeak.RabbitMQ.Interfaces;
+
+namespace EasySpeak.Notifier.WebAPI.Services
+{
+    public class ConsumerHostedService : BackgroundService
+    {
+        private readonly IMessageConsumer _consumer;
+        public ConsumerHostedService(IMessageConsumer consumer) 
+        { 
+            _consumer = consumer;
+        }
+
+        public override void Dispose()
+        {
+            base.Dispose();
+        }
+
+        public override async Task StartAsync(CancellationToken cancellationToken)
+        {
+            await base.StartAsync(cancellationToken);
+        }
+
+        public override async Task StopAsync(CancellationToken cancellationToken)
+        {
+            await base.StopAsync(cancellationToken);
+        }
+
+        protected override Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+                try
+                {
+                    //var hubContext = (IHubContext) app.ApplicationServices.GetService<IHubContext>()!;
+                    _consumer.Recieve<string>("notifier", (data) =>
+                    {
+                        Console.WriteLine(data);
+                        //await hubContext.Clients.All.SendAsync("user", data);
+                    });
```

Comment:
why do you comment hub?

Reply:
Because it is not a part of task, it is just an example how we can work with hub

Reply:
yes, but we will need this example in the future